src/duties/library.js

```diff
@@ -1,10 +1,7 @@
 let _ = require('lodash/fp')
 
 let emptyChangelog = ({ danger, fail }) => {
-  if (
-    !_.includes('CHANGELOG.md', danger.git.modified_files) &&
-    !_.includes('CHANGELOG.md', danger.git.added_files)
-  ) {
+  if (!/changelog/i.test([danger.git.modified_files, danger.git.added_files])) {
```

Comment:
You prob want `/^changelog`

Reply:
hm, it doesn't like `/^changelog` 🤔

`/\bchangelog` should be safe - I'll use that instead

Reply:
Ok then